tests/kernel/update_kernel.pm

```diff
@@ -28,12 +28,14 @@ use package_utils;
 sub check_kernel_package {
     my $kernel_name = shift;
 
+    enter_trup_shell(global_options => '-c') if is_transactional;
```

Comment:
If is system rebooted , function works even on transactional system and there is no change needed.  If i remember correctly we decided to do reboot when we introduced support for transactional systems in `update_kernel`. 

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/7eefe7bbf87329880a2cf786271bf1b6aa174380/tests/kernel/update_kernel.pm#L114

But we can proceed with your suggestion.

Reply:
`check_kernel_package()` is called from multiple places, some of which reboot *after* the check.